static/js/components/widgets/SelectField.tsx

```diff
@@ -82,7 +87,8 @@ export default function SelectField(props: Props): JSX.Element {
         border:    0,
         boxShadow: "none"
       })
-    }
+    },
+    ...(isOptionDisabled ? { isOptionDisabled } : {})
```

Comment:
here too, the props match so why not just put the value into the object, whatever it might be? 

```suggestion
    },
    isOptionDisabled
```